pyaedt/circuit.py

```diff
@@ -178,18 +178,18 @@ class Circuit(FieldAnalysisCircuit, object):
         nothing is used.
     specified_version: str, optional
         Version of AEDT to use. The default is ``None``, in which case
-        the active version or latest installed version is  used.
+        the active version or latest installed version is  used. This parameter is ignored when Script is launched within AEDT.
```

Comment:
These lines have to be wrapped.  PEP8 says 79 char for code, 72 for docstring.  We should be ok going a little over, but this is 127 char and won't display well if the user runs `help(...)`
https://www.python.org/dev/peps/pep-0008/#id19